test/token-locking.js

```diff
@@ -0,0 +1,179 @@
+/* globals artifacts */
+
+import { forwardTime, getTokenArgs, checkErrorRevert } from "../helpers/test-helper";
+
+const EtherRouter = artifacts.require("EtherRouter");
+const IColonyNetwork = artifacts.require("IColonyNetwork");
+const IColony = artifacts.require("IColony");
+const ITokenLocking = artifacts.require("ITokenLocking");
+const Token = artifacts.require("Token");
+
+contract("TokenLocking", addresses => {
+  let token;
+  let tokenLocking;
+  let otherToken;
+  let colonyNetwork;
+  let colony;
+
+  before(async () => {
+    const etherRouter = await EtherRouter.deployed();
+    colonyNetwork = IColonyNetwork.at(etherRouter.address);
+    const tokenLockingAddress = await colonyNetwork.getTokenLocking.call();
+    tokenLocking = ITokenLocking.at(tokenLockingAddress);
+  });
+
+  describe("when locking tokens", async () => {
+    const usersTokens = 6;
+
+    beforeEach(async () => {
+      let tokenArgs = getTokenArgs();
+      token = await Token.new(...tokenArgs);
+      await token.mint(usersTokens);
+
+      tokenArgs = getTokenArgs();
+      otherToken = await Token.new(...tokenArgs);
+
+      const { logs } = await colonyNetwork.createColony(token.address);
+      const { colonyAddress } = logs[0].args;
+      colony = IColony.at(colonyAddress);
+    });
+
+    it("should correctly set colony network address", async () => {
+      await tokenLocking.setColonyNetwork(0x0);
+      let colonyNetworkAddress = await tokenLocking.getColonyNetwork.call();
+      assert.equal(colonyNetworkAddress, 0x0);
+
+      await tokenLocking.setColonyNetwork(colonyNetwork.address);
+      colonyNetworkAddress = await tokenLocking.getColonyNetwork.call();
+      assert.equal(colonyNetworkAddress, colonyNetwork.address);
+    });
+
+    it("should correctly deposit tokens", async () => {
+      await token.approve(tokenLocking.address, usersTokens);
+      await tokenLocking.deposit(token.address, usersTokens);
+      const userDepositedBalance = await tokenLocking.getUserLockedBalance.call(token.address, addresses[0]);
```

Comment:
Can you change the account used in all tests here as the user whose tokens get locked to be another account than the coinbase. This is the default account which owns most of the contracts you test with so it's risky to use it as a representative account for testing permissions especially. Shouldn't be that difficult to switch to `addresses[1]` for example.